test/upload-file-test.js

```diff
@@ -187,6 +188,57 @@ suite.addBatch({
                                     assert.ifError(err);
                                 }
                             },
+                            "and we get the file from web interface without login": {
```

Comment:
"the web interface" instead of "web interface"; "logging in" instead of "login"

Reply:
Done,